blocks/header/header.css

```diff
@@ -186,6 +212,62 @@ header nav .nav-sections ul > li > ul > li {
   font-weight: 500;
 }
 
+header .nav-sections > ul > li:first-child {
+  display: none;
+}
+
+@media (min-width: 900px) {
+  header .nav-sections > ul > li:first-child {
+    display: block;
+    background: url("/icons/checkin.svg") no-repeat;
+    text-indent: 23px;
+  }
+}
+
+header .nav-sections > ul > li:nth-child(2) {
+  background: url("/icons/guest_icon_white.svg") no-repeat;
+  background-size: cover;
+  text-indent: -1000px;
+  height: 24px;
+  width: 21px;
+  position: absolute;
+  right: 112px;
+  top: 28px;
+}
+
+header .nav-sections > ul > li:nth-child(3) {
+  background: url("/icons/phone.svg") no-repeat;
+  background-size: cover;
+  text-indent: -1000px;
+  height: 24px;
+  width: 27px;
+  position: absolute;
+  right: 72px;
+  top: 28px;
+}
+
+@media (min-width: 900px) {
+  header .nav-sections > ul > li:nth-child(2) {
+    background-size: unset;
+    text-indent: 23px;
+    height: unset;
+    width: unset;
+    position: unset;
+    right: unset;
+    top: unset;
+  }
+  
+  header .nav-sections > ul > li:nth-child(3) {
+    background-size: unset;
+    text-indent: 23px;
+    height: unset;
+    width: unset;
+    position: unset;
+    right: unset;
+    top: unset;
+  }
+}
+
 @media (min-width: 900px) {
```

Comment:
breakpoint on the original site is 992px
we could standardize at 900 which would be better actually but we need to align between us and then with customer because the rest of the site is still on the old platform.
Sould suggest to keep 992px now. wdyt?

Reply:
Yeah, I kinda went with the default Franklin one (600, 900 & 1200). But for now it would be a safer bet to go ahead with the ones customer has. I will update this.